core/vm/evm.go

```diff
@@ -453,7 +453,7 @@ func (evm *EVM) create(caller ContractRef, codeAndHash *codeAndHash, gas uint64,
 	snapshot := evm.StateDB.Snapshot()
 	evm.StateDB.CreateAccount(address)
 	if evm.chainRules.IsEIP158 {
-		evm.StateDB.SetNonce(address, 1)
+		evm.StateDB.SetNonce(address, 1) // Need to keep this in mind
```

Comment:
Wondering what this comment means

Reply:
The nonce of a contract address is set to 1. This is one reason that it may be a bad idea to have the zero address deploy a contract.